lib/assignDisabledRanges.mjs

```diff
@@ -268,34 +345,58 @@ export default function assignDisabledRanges(root, result) {
 				continue;
 			}
 
-			throw comment.error(`"${ruleToEnable}" has not been disabled`, {
+			throw node.error(`"${ruleToEnable}" has not been disabled`, {
 				plugin: 'stylelint',
 			});
 		}
 	}
 
 	/**
-	 * @param {PostcssComment} comment
+	 * @param {PostcssNode} node
```

Comment:
minor

The type could be more specific.
e.g. `{AtRule|Rule|Declaration}`

Reply:
Is there a need to make it more specific?

We are using this value to throw errors `PostCSS`, the common type that can throw errors is `postcss.Node`.

Reply:
> Is there a need to make it more specific?

It sometimes help to surface potential bugs down the line.
It's minor anyway.